src/main/scala/no/ndla/learningpathapi/controller/LearningpathController.scala

```diff
@@ -543,6 +547,20 @@ trait LearningpathController {
       params.get(paramName).map(_.trim).filterNot(_.isEmpty())
     }
 
+    def paramAsListOfLong(paramName: String)(implicit request: HttpServletRequest): List[Long] = {
+      params.get(paramName) match {
+        case None => List()
+        case Some(param) => {
+          val paramAsListOfStrings = param.split(",").toList.map(_.trim)
+          if (!paramAsListOfStrings.forall(entry => entry.forall(_.isDigit))) {
+            throw new ValidationException(errors = List(ValidationMessage(paramName, s"Invalid value for $paramName. Only (list of) digits are allowed.")))
```

Comment:
Det vil ikke fungere på samme måte som vi gjør i controlleren i article-api, der errorhandleren returnerer en `ActionResult`. I dette tilfellet ville da det resultatet bli satt på variabelen som gjorde kallet til `paramAsListOfLong`. 

Denne metoden kunne muligens returnert en `Try` i stedet for å kaste en exception, men klarer ikke finne en måte å gjøre bruken av metoden pen da. Det fører til unødig kompleksitet i `get("/")...`